frontend/src/views/TemplateView.jsx

```diff
@@ -66,16 +72,18 @@ export const TemplateView = ({ endpoint }) => {
         </div>
 
         <main>
-          <div className="w-[100vw] overflow-y-auto">
-            <div className="relative ml-12 mr-12 flex flex-1 ">
-              <div className="html-wrapper relative flex h-screen items-center justify-center pt-28">
-                {!!!objectID && !!!singleObjectID && (
-                  <p
-                    className={`${isSmallScreen ? "ml-32" : "ml-80"} text-xl text-gray-700 dark:text-gray-300 `}
-                  >
-                    Select an Instance
-                  </p>
-                )}
+          <div className="overflow-y-auto md:w-[100vw]">
+            {!!!objectID && !!!singleObjectID && (
```

Comment:
```suggestion
            {!objectID && !singleObjectID && (
```

In this case, triple exclamation marks should be exactly the same.